### PR TITLE
Improve installation instructions and add contributing guide

### PR DESCRIPTION
**Pull request by JustinyAhin**
This PR add details to installations intructions, and a contributing guide.

### CODE REVIEW

**Comment by JustinyAhin** [comment]
Overall, do we need a specific CONTRIBUTING.md for the guidelines?

File: README.md [hunk 1]
```diff
@@ -35,3 +43,27 @@ This command generates static content into the `build` directory and can be serv
...
+## Contributing to Yoast Developer portal
+Howdy! It's really great that you want to contribute to the Yoast developer portal. The development of the portal takes place on GitHub.
+
+There are three main ways to contribute:
+1. [Contributing to issues](#contributing-to-issues)
+2. [Reviewing pull requests](#reviewing-pull-requests)
+3. [Contributing to code](#contributing-to-code)
+
+### Contributing to issues
+We use Jira to track changes and improvements to the portal that need to be discussed and completed. The portal related tickets are [in this board](#).
```
Comment (moorscode) [inline README.md:57]: Jira isn't open for people outside of the company, so I guess the description needs to just ask for a GitHub issue on this repository.

File: README.md [hunk 2]
```diff
@@ -35,3 +43,27 @@ This command generates static content into the `build` directory and can be serv
...
+
+Before creating a ticket, please check if the issue you want to raise have not already received a ticket.
+
+Please be as descriptive and succinct as possible in your ticket, and if necessary, to help better understand the issue, add screenshots or video captures.
+
+### Reviewing pull requests
```
Comment (moorscode) [inline README.md:63]: Not sure if you wanted to add information here, having a header without copy seems weird :)

File: README.md [hunk 3]
```diff
@@ -35,3 +43,27 @@ This command generates static content into the `build` directory and can be serv
...
+
```
Comment (moorscode) [inline README.md:64]:
```suggestion
```

**Review by moorscode — CHANGES_REQUESTED** [review]
Some small things that stand out :)

**Review by moorscode — APPROVED** [review]

**Comment by moorscode** [comment]
:+1:

**Pull request merged**